plugins/feeds/public/hybrid_analysis.py

```diff
@@ -116,29 +118,38 @@ def analyze(self, item):
                 sha256_new_file = sha256.SHA256(value=extracted_file["sha256"]).save()
 
                 new_file.link_to(sha256_new_file, "sha256", self.name)
+                
+                path_extracted_file = None
+                if extracted_file["file_path"] and isinstance(extracted_file["file_path"], str):
```

Comment:
Didn't you mean to check the presence of `file_path` in `extracted_file` here like in other changes?